app/build.gradle

```diff
@@ -77,6 +75,9 @@ dependencies {
     testImplementation 'com.nhaarman:mockito-kotlin-kt1.1:1.5.0'
     testImplementation 'org.amshove.kluent:kluent:1.31'
 
+    // Leak Canary
+    debugImplementation "com.squareup.leakcanary:leakcanary-android:${versions.leak_canary}"
```

Comment:
Where's release implementation?

Reply:
they removed it recently on march 30th. It is just a single implementation for all now
Reference : https://github.com/square/leakcanary/issues/979#issuecomment-478262462